src/header/css/header-menus.pcss

```diff
@@ -438,7 +443,10 @@
 			width: calc(350px * 2 + 20px + 35px * 2);
 
 				&[data-structuredcols=""] {
-					width: calc(300px * 2 + 20px + 20px * 2);
+					width: calc(300px * 2 + 20px + 35px * 2);
```

Comment:
seems weird to use calc for a value that you could just insert here as px?

Reply:
Just following the existing pattern @paulcpederson